client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -94,24 +65,82 @@ function getHoldMessages( translate: LocalizeProps[ 'translate' ] ) {
 	};
 }
 
+function getBlockingMessages( translate: LocalizeProps[ 'translate' ] ) {
+	return {
+		BLOCKED_ATOMIC_TRANSFER: {
+			message: translate(
+				'This site is not currently eligible to install themes and plugins. Please contact our support team for help.'
+			),
+			status: 'is-error',
+			contactUrl: localizeUrl( 'https://wordpress.com/help/contact' ),
+		},
+		TRANSFER_ALREADY_EXISTS: {
+			message: translate(
+				'Installation in progress. Just a minute! Please wait until the installation is finisehd, then try again.'
+			),
+			status: null,
+			contactUrl: null,
+		},
+		NO_JETPACK_SITES: {
+			message: translate( 'Try using a different site.' ),
+			status: 'is-error',
+			contactUrl: null,
+		},
+		NO_VIP_SITES: {
+			message: translate( 'Try using a different site.' ),
+			status: 'is-error',
+			contactUrl: null,
+		},
+		SITE_GRAYLISTED: {
+			message: translate(
+				"There's an ongoing site dispute. Contact us to review your site's standing and resolve the dispute."
+			),
+			status: 'is-error',
+			contactUrl: localizeUrl( 'https://en.support.wordpress.com/suspended-blogs/' ),
+		},
+		NO_SSL_CERTIFICATE: {
+			message: translate(
+				'Certificate installation in progress. Hold tight! We are setting up a digital certificate to allow secure browing on your site using "HTTPS".'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Hold tight! We are setting up a digital certificate to allow secure browsing on your site, using "HTTPS". Please try again in a few minutes.' )` *ES Score: 6*